public/about_us/about.html

```diff
@@ -629,6 +629,47 @@ <h1 class="headline">Meet The Team</h1>
 								</p>
 							</div>
 						</div>
+						<div class="person">
+							<div class="display-picture">
+								<img data-src="./members/Ken Fahmidur Rahman.jpg" src="icons/loading.svg" alt="">
+							</div>
+							<div class="content">
+								<div class="name"><span>Ken Fahmidur Rahman</span></div>
+								<div class="bio">
+									<p>Electrical Engineering in the making. He hopes to switch to Data Science in pursuit of Higher Education...
+										<span>read more</span>
+									</p>
+								</div>
+							</div>
+							<!-- hidden content for serving data for detailed pop-up -->
+							<div class="details hide">
+								<span class="designation">Data analyst</span>
+								<span class="project-name"></span>
+								<div class="social-links">
+									<span class="facebook"></span>
+									<span class="twitter"></span>
+									<span class="email"></span>
+									<span class="instagram"></span>
+								</div>
+								<div class="info">
+									<div class="line">
+										<span class="title">Education:</span>
+										<p class="data">Undergraduate Student 4th year Military Institute of Science and Technology, Dhaka, Bangladesh</p>
+									</div>
+									<div class="line">
+										<span class="title">Current Workplace(s):</span>
+										<p class="data">N/A</p>
```

Comment:
Replace N/A with empty string. We dont want current workplaces to show if theres no valid data.